site/content/docs/5.2/components/close-button.md

```diff
@@ -45,6 +45,38 @@ Close buttons can also be created without `.btn-close`.
   <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false"><use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"></use></svg>
   <span class="visually-hidden">Close</span>
 </button>
+
+<button type="button" class="btn btn-icon btn-secondary">
+  <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false"><use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"></use></svg>
+  <span class="visually-hidden">Close</span>
+</button>
+
+<button type="button" class="btn btn-icon btn-secondary" disabled>
+  <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false"><use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"></use></svg>
+  <span class="visually-hidden">Close</span>
+</button>
+{{< /example >}}
+
+{{< example class="bg-dark" >}}
+<button type="button" class="btn btn-icon btn-no-outline btn-inverse">
+  <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false"><use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"></use></svg>
+  <span class="visually-hidden">Close</span>
+</button>
+
+<button type="button" class="btn btn-icon btn-no-outline btn-inverse" disabled>
```

Comment:
There is an issue here, it should be grayish instead of white (like this one `<button type="button" class="btn btn-icon btn-inverse btn-secondary" disabled>`). Otherwise we can't differentiate it from a regular button.

Reply:
I think the main issue is that `.btn-no-outline` doesn't have a disabled state for now but it might be fixed via #1426.

Reply:
Done